tutor/src/models/course/builder-ux.js

```diff
@@ -178,7 +178,7 @@ export default class CourseBuilderUX extends BaseModel {
 
   // skips
   skip_new_or_copy() {
-    return !!this.source;
+    return !!this.source || !Courses.nonPreview.size;
   }
```

Comment:
FYI I also added `any`  and `isEmpty` boolean checks.  Could be `Courses.nonPreview.isEmpty`